### PR TITLE
Form Introduction

### PR DESCRIPTION
**Pull request by AK-Devil**
I added new section in main index.html page. The section in about comment section. It have 5 input field and a button. Its also check html validation.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -304,13 +303,43 @@ <h2>A Special Time</h2><br />
...
+      <fieldset>
+        <legend>comment</legend><br />
+        <label class="margin" for="User_name">Name: </label>  
+        <input class="input-field" type="text" id="User_name" name="User_name" maxlength="20" required><br><br>
+        
+        <label for="email">Email: </label>
+        <input class="input-field" type="email" id="email" name="email" maxlength="20" required><br><br>
+        
+        <label for="comment">Comment</label><br />
+        <textarea class="textfield" id="comment"  name="comment" placeholder="Type your comment here…" rows=5 maxlength="250" required></textarea>
```
Comment (NoTengoBattery) [inline index.html:316]: Good work, all labels are in place.

File: index.html [hunk 2]
```diff
@@ -304,13 +303,43 @@ <h2>A Special Time</h2><br />
...
+        <br />
+          <fieldset >
+            <legend style="margin: 0 auto;">What is your overall opinion about this article?</legend>
+              <input type="radio" id="position" name="opinion" value="position" checked>
+              <label for="position">Position</label><br>
+              <input type="radio" id="negative" name="opinion" value="negative">
+              <label for="negative">Negative</label><br>
+              <input type="radio" id="neutral" name="opinion" value="neutral">
+              <label for="neutral">Neutral</label>
```
Comment (NoTengoBattery) [inline index.html:325]: You have the radio button fieldset, but your "positive" option is misspelled as "position".

**Comment by NoTengoBattery** [comment]
Code review
===

- [ ] You have linter errors in an unrelated file. Deleting the file might work.
- [ ] Your pull request does not tell anything about your UX design decisions. You can always edit your pull request message at any time.
- [ ] You did not change the font family of the form.

Good work, overall. Please take care of the pull request messages, they are really important.

**Comment by AK-Devil** [comment]
clear all errors and change the font family of comment section

**Comment by NoTengoBattery** [comment]
@AK-Devil Okay, everything looks good.

**Pull request closed without merging**